### PR TITLE
OSDOCS-8464: add release notes file to MicroShift 4.15 branch

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.15 

Issue:
https://issues.redhat.com/browse/OSDOCS-8464

Link to docs preview:

QE review:
- N/A, docs plumbing

Additional information:

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@JoeAldinger Could you also PTAL?

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,241 @@
+:_mod-docs-content-type: ASSEMBLY
```
Comment (JoeAldinger) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:1]: Nice

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+This section is updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {microshift-short} {ocp-version}. Versioned asynchronous releases, for example with the form {microshift-short} {ocp-version}.z, will be detailed in the following subsections.
+
+//FIXME-TODO
+[id="microshift-4-15-0-dp"]
+=== RHSA-2024:XXXX - {microshift-short} 4.15.0 bug fix and security update advisory
+
+Issued: 2024-02-28
+
+{product-title} release 4.14.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:XXXX[RHSA-2024:XXXX] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:XXXX[RHSA-2024:XXXX] advisory.
```
Comment (JoeAldinger) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:238]:
```suggestion
{product-title} release 4.15.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:XXXX[RHSA-2024:XXXX] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:XXXX[RHSA-2024:XXXX] advisory.
```

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+* _Available_
+
+* _Deprecated_
+
+* _Removed_
+
+.{product-title} deprecated and removed features tracker
+[cols="5,1,1,1,1",options="header"]
+|====
+|Feature |4.12 |4.13 |4.14 |4.15
```
Comment (JoeAldinger) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:129]:
```suggestion
|Feature |4.13 |4.14 |4.15
```

Comment (JoeAldinger) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:129]: In core we only do the last 3 releases

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+
+[id="microshift-4-14-updates-supported"]
+==== Updates are supported on 4.14 and later
+The following list provides update details:
+
+* {microshift-short} offers in-place updates on {op-system-ostree} systems with automatic system rollback capabilities and automatic back up and restore functions.
+* Updates of the RPMs on a non-OSTree system such as {op-system} are also supported.
+* Updates from preview versions such as {microshift-short} 4.13 and earlier are not supported.
+
+//[id="microshift-4-14-new-feat-based-on-{op-system-ostree}"]
```
Comment (JoeAldinger) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:55]:
```suggestion
//[id="microshift-4-15-new-feat-based-on-{op-system-ostree}"]
```

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+
+** If you run third-party monitoring and security agents that depend on the cgroup file system, update the agents to versions that support cgroup v2.
+** If you run cAdvisor as a standalone DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
+** If you deploy Java applications with the JDK, ensure you are using JDK 11.0.16 and later or JDK 15 and later, which fully support cgroup v2.
+
+[id="microshift-4-15-updating"]
+=== Updating
+Updates for both minor releases and patch releases are supported.
+
+[id="microshift-4-14-updates-supported"]
```
Comment (JoeAldinger) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:47]:
```suggestion
[id="microshift-4-15-updates-supported"]
```

Comment (JoeAldinger) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:47]: I think you want 4.15 here.

**Review by JoeAldinger — COMMENTED** [review]
small stuff, I wasn't sure how much you wanted to carryover and what should be // out, but it otherwise looks good.

**Review by jeana-redhat — APPROVED** [review]
LGTM
/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**